config/config.go

```diff
@@ -109,6 +109,17 @@ var (
 	DefaultMarathonSDConfig = MarathonSDConfig{
 		RefreshInterval: Duration(30 * time.Second),
 	}
+
+	// The default Kubernetes SD configuration
+	DefaultKubernetesSDConfig = KubernetesSDConfig{
+		NodeMetricsPort: 10255,
+		ScrapeKey:       "prometheus.io/scrape",
```

Comment:
These are your annotations, not everyones. To be generic we should allow this to be configurable by relabelling.